### ISSUE TITLE
bug: incompatibility with AzAPI v1.13

### ISSUE THREAD

**Issue by matt-FFFFFF** [Azure/terraform-azurerm-lz-vending#369]
We are aware of an issue with the latest minor release of AzAPI v1.13. This has been caused by the provider making breaking changes.

For now we recommend customers add this code to their root module:

```terraform
terraform {
  required_providers {
    azapi = {
      source  = "azure/azapi"
      version = "< 1.13"
    }
  }
}
```

**Comment by JoshuaMcConnaughey** [comment]
Hello @matt-FFFFFF,

Any update on when fix #370 will be available? I didn't realize my azapi provider version wasn't pinned, and it upgraded to v1.13.1. Doesn't seem like I can run a successful apply, and I can't downgrade now that my resources are marked as having been managed by the newer version.

Thank you!

**Comment by stevexiong-hennepin** [comment]
Any updates when this? Thanks.

**Comment by matt-FFFFFF** [comment]
With v1.13.1 this is now fixed

**Comment by matt-FFFFFF** [comment]
> Hello @matt-FFFFFF,
> 
> Any update on when fix #370 will be available? I didn't realize my azapi provider version wasn't pinned, and it upgraded to v1.13.1. Doesn't seem like I can run a successful apply, and I can't downgrade now that my resources are marked as having been managed by the newer version. Any ideas for how to proceed until the fix is available?
> 
> Thank you!

We will now put off this update until v2 of AzAPI, rather than making a breaking change and introducing a minimum of v1.13.1
